benchmarks/external/workerpoolWorker.js

```diff
@@ -12,5 +12,5 @@ function yourFunction (data) {
 }
 
 workerpool.worker({
-  yourFunction: yourFunction
+  yourFunction
```

Comment:
The object key is required in plain JS, TS add it and transpilation. The workerpool will fail with that change.